### PR TITLE
Add more detail to intro and abstract.

### PR DESCRIPTION
**Pull request by daffidwilde**
Following comments from DMKD, more detail has been added to the introduction and abstract around the methods used and the purpose of the analysis in the closing sections.

### CODE REVIEW

File: arxiv/main.tex [hunk 1]
```diff
@@ -45,15 +45,18 @@
...
+    find the set of initial cluster centroids, is compared with the original
+    initialisation for \(k\)-modes as well as the next most popular method. In
+    order to highlight the merits of the proposed method two stages of analysis
+    are presented. The first uses benchmark datasets to give familiarity whilst
+    the second utilises a body of artificial datasets that were generated to
+    reveal the nuances of each method. Based on this analysis, the proposed
+    method is shown to outperform the other initialisations in the majority of
+    cases, especially when the number of clusters is optimised. In addition, the
+    proposed method is found to outperform the popular established method
+    specifically for low-density data.
```
Comment (gillardjw) [inline arxiv/main.tex]: I prefer the style and sentences you've got in the introduction, I would specifically mention Huang and Cao in the abstract, and mention the analysis that you conclude the paragraph in the introduction with, I'd be more specific, and there is no harm in repeating sentences between the abstract and introduction. I think I'd also get rid of "mathematical fairness". That's too vague a concept, What about prior knowledge? Something like that?

File: arxiv/main.tex [hunk 2]
```diff
@@ -45,15 +45,18 @@
...
     not. The method, which utilises the Hospital-Resident Assignment Problem to
-    find the set of initial cluster centroids, is compared with the current
-    initialisations on both benchmark datasets and a body of newly generated
-    artificial datasets. Based on this analysis, the proposed method is shown to
-    outperform the other initialisations in the majority of cases, especially
-    when the number of clusters is optimised. In addition, we find that our
-    method outperforms the leading established method specifically for
-    low-density data.
+    find the set of initial cluster centroids, is compared with the original
+    initialisation for \(k\)-modes as well as the next most popular method. In
```
Comment (drvinceknight) [inline arxiv/main.tex]:
```suggestion
    find the set of initial cluster centroids, is compared with two
    initialisation methods for \(k\)-modes~\cite{Cao2009} as well as the 
    next most popular method present in the literature~\cite{Huang1998}. In
```

File: arxiv/main.tex [hunk 3]
```diff
@@ -45,15 +45,18 @@
 \begin{abstract}
     This paper presents a new way of selecting an initial solution for the
     \(k\)-modes algorithm that allows for a notion of mathematical fairness and
```
Comment (drvinceknight) [inline arxiv/main.tex]:
```suggestion
    \(k\)-modes algorithm that allows for a notion of game theoretic fairness and
```

how do you feel about that as a suggestion @gillardjw re your point about vagueness of `mathematical fairness`? I don't disagree that that's vague but perhaps `game theoretic` is slightly more precise?

File: arxiv/main.tex [hunk 4]
```diff
@@ -66,9 +69,19 @@ \section{Introduction}\label{sec:intro}
 the quality of its initial solution. The initialisation method proposed in this
 work extends the method presented by Huang~\cite{Huang1998} by using results
 from game theory to ensure mathematical fairness and to lever the full learning
-opportunities presented by the data being clustered. In doing so, it is
-demonstrated that the proposed method is able to outperform both of the
-established initialisations for \(k\)-modes. The paper is structured as follows:
+opportunities presented by the data being clustered. In addition to the method
```
Comment (drvinceknight) [inline arxiv/main.tex]: Same comments (I see that I've made my suggestions on the arXiv version).

**Pull request merged**